06.bowling_object/lib/game.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_relative './frame'
+require_relative './last_frame'
+require_relative './shot'
+
+class Game
+  def initialize
+    @frames = Array.new(10) do |index|
+      index == 9 ? LastFrame.new(index) : Frame.new(index)
+    end
+  end
+
+  def play(marks)
+    copied_marks = marks.dup
+    @frames.each do |frame|
+      loop do
+        break if frame.full?
```

Comment:
```suggestion
      until frame.full?
```

untilにしなかった理由って何かありますか？（修正漏れ？）

Reply:
修正漏れでした。。